src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Main {
+    static final int COUNT_OF_FIGURES = 5;
+
+    public static void main(String[] args) {
+        FigureSupplier figuresupplier = new FigureSupplier();
+        Figure [] figures = new Figure[COUNT_OF_FIGURES];
+
+        for (int i = 0; i < COUNT_OF_FIGURES; i++) {
+            figures[i] = figuresupplier.getRandomFigure();
+        }
+        for (int i = 0; i < COUNT_OF_FIGURES; i++) {
+            figures[i].draw();
+        }
+
+        Figure a = figuresupplier.getDefaultFigure();
+        a.draw();
```

Comment:
The creation and drawing of the default figure could be part of the loop that handles the random figures, or you can separate the logic for creating and drawing figures into different methods for better code organization and readability.